CI/travis/get_ldist

```diff
@@ -0,0 +1,35 @@
+get_ldist() {
+	case "$(uname)" in
+	Linux*)
+		if [ ! -f /etc/os-release ] ; then
```

Comment:
why this if block ?
is there a case where this happens ?
I admit I don't have a CentOS 6 VM to check ; I only have a CentOS 7 around

Reply:
Darwin doesn't support /etc/os-release - that's a Linux only thing.

Reply:
centos 6 doesn't include /etc/os-release, they only added that in centos 7